barretenberg/cpp/src/barretenberg/honk/transcript/goblin_ultra_transcript.test.cpp

```diff
@@ -0,0 +1,233 @@
+    {
+        TranscriptManifest manifest_expected;
+
+        auto log_n = numeric::get_msb(circuit_size);
+
+        size_t max_relation_length = Flavor::MAX_RANDOM_RELATION_LENGTH;
+        size_t size_FF = sizeof(FF);
+        size_t size_G = 2 * size_FF;
+        size_t size_uni = max_relation_length * size_FF;
+        size_t size_evals = (Flavor::NUM_ALL_ENTITIES)*size_FF;
+        size_t size_uint32 = 4;
+
+        size_t round = 0;
+        manifest_expected.add_entry(round, "circuit_size", size_uint32);
+        manifest_expected.add_entry(round, "public_input_size", size_uint32);
+        manifest_expected.add_entry(round, "pub_inputs_offset", size_uint32);
+        manifest_expected.add_entry(round, "public_input_0", size_FF);
+        manifest_expected.add_entry(round, "W_L", size_G);
+        manifest_expected.add_entry(round, "W_R", size_G);
+        manifest_expected.add_entry(round, "W_O", size_G);
+        manifest_expected.add_entry(round, "ECC_OP_WIRE_1", size_G);
+        manifest_expected.add_entry(round, "ECC_OP_WIRE_2", size_G);
+        manifest_expected.add_entry(round, "ECC_OP_WIRE_3", size_G);
+        manifest_expected.add_entry(round, "ECC_OP_WIRE_4", size_G);
+        manifest_expected.add_challenge(round, "eta");
+
+        round++;
+        manifest_expected.add_entry(round, "SORTED_ACCUM", size_G);
+        manifest_expected.add_entry(round, "W_4", size_G);
+        manifest_expected.add_challenge(round, "beta", "gamma");
+
+        round++;
+        manifest_expected.add_entry(round, "Z_PERM", size_G);
+        manifest_expected.add_entry(round, "Z_LOOKUP", size_G);
+        manifest_expected.add_challenge(round, "Sumcheck:alpha", "Sumcheck:zeta");
+
+        for (size_t i = 0; i < log_n; ++i) {
+            round++;
+            std::string idx = std::to_string(i);
+            manifest_expected.add_entry(round, "Sumcheck:univariate_" + idx, size_uni);
+            std::string label = "Sumcheck:u_" + idx;
+            manifest_expected.add_challenge(round, label);
+        }
+
+        round++;
+        manifest_expected.add_entry(round, "Sumcheck:evaluations", size_evals);
+        manifest_expected.add_challenge(round, "rho");
+
+        round++;
+        for (size_t i = 0; i < log_n; ++i) {
+            std::string idx = std::to_string(i);
+            manifest_expected.add_entry(round, "ZM:C_q_" + idx, size_G);
+        }
+        manifest_expected.add_challenge(round, "ZM:y");
+
+        round++;
+        manifest_expected.add_entry(round, "ZM:C_q", size_G);
+        manifest_expected.add_challenge(round, "ZM:x", "ZM:z");
+
+        round++;
+        // TODO(Mara): Make testing more flavor agnostic so we can test this with all flavors
+        manifest_expected.add_entry(round, "ZM:PI", size_G);
+        manifest_expected.add_challenge(round); // no challenge
+
+        return manifest_expected;
+    }
+
+    void generate_test_circuit(auto& builder)
```

Comment:
mostly copied from generate_test_circuit in goblin_ultra_composer.test.cpp, but with only 1 big add gate.